contracts/globalConstraints/EtherGC.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity 0.5.17;
+
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "./GlobalConstraintInterface.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../controller/Avatar.sol";
+
+
+/**
+ * @title EtherGC ether constraint per period
+ */
+contract EtherGC is GlobalConstraintInterface {
+    using SafeMath for uint256;
+
+    uint256 public periodLength; //the period length in blocks units
+    uint256 public amountAllowedPerPeriod;
```

Comment:
Not sure if this should be percentage-wise, or absolute values. Probably better to keep it absolute.